instrumentation/kafka-clients-0.11/src/main/java/io/opentelemetry/auto/instrumentation/kafka_clients/KafkaDecorator.java

```diff
@@ -54,24 +53,35 @@ protected String getComponentName() {
     return "java-kafka";
   }
 
-  public void onConsume(final Span span, final ConsumerRecord record) {
+  public String spanNameOnConsume(final ConsumerRecord record) {
+    final String topic = record.topic();
+    if (topic != null) {
+      return topic;
+    } else {
```

Comment:
this else is redundant, since the if has a `return` in it.

Reply:
I actually like this flow personally. Do we have a style guide that we can refer to for these kinds of style decisions?

Reply:
It doesn't look like the google style guide has anything to say on the subject. Up to you, I guess.

Reply:
Could also replace with a ternary operator.